client/src/DbtLanguageClient.ts

```diff
@@ -50,6 +52,9 @@ export class DbtLanguageClient implements Disposable {
       this.client.onNotification('custom/updateQueryPreviewDiagnostics', ({ uri, diagnostics }) => {
         this.previewContentProvider.updateDiagnostics(uri as string, diagnostics as Diagnostic[]);
       }),
+      this.client.onNotification('custom/languageServerEventNotification', (event: DebugEvent) => {
```

Comment:
Can we rename it to just 'custom/manifestParsed' and get rid of parameters? This the real purpose of this event. Let's do as simple as possible.

Reply:
Seems not. This event is more about LS events - not manifest.